src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -308,25 +308,24 @@ namespace
                       m_text_project_filepath->SetText(project_file);
                       return TRUE;
                     }
-
                   default:
-                    return FALSE;
+                  return FALSE;
                 }
 
               case WM_COMMAND:
                 switch (LOWORD(wparam))
                 {
                   case IDC_BUTTON_BROWSE:
                     {
-                        MSTR filepath;
-                        if (get_save_project_filepath(hwnd, filepath))
-                        {
-                            map->GetParamBlock()->SetValueByName(L"project_path", filepath, 0);
-                            ICustEdit* project_path = GetICustEdit(GetDlgItem(hwnd, IDC_TEXT_PROJECT_FILEPATH));
-                            project_path->SetText(filepath);
-                            ReleaseICustEdit(project_path);
-                        }
-                        return TRUE;
+                      MSTR filepath;
```

Comment:
Revert indentation.

Reply:
I looked up how it was formatted in the past and reverted it.